bin/docker_build

```diff
@@ -22,6 +23,19 @@ if [ "$1" == "inline" ]; then
     shift
 fi
 
+if [ "$1" == build ]; then
+    build=y
+    shift
+fi
+
+path=
+if [ "$1" == "tar" ]; then
```

Comment:
I think tar can be a separate step. It's complicating all the options in cmd/build and bin/docker_build and the option is only used on github actions.

Reply:
Moved compressing actions out of cmd/build and docker build